src/bun.js/node/node_fs.zig

```diff
@@ -3442,6 +3443,34 @@ pub const NodeFS = struct {
                 const fd = switch (args.path) {
                     .path => brk: {
                         path = args.path.path.sliceZ(&this.sync_error_buf);
+                        if (this.vm) |vm| {
+                            if (vm.standalone_module_graph) |graph| {
+                                if (graph.find(path)) |file| {
+                                    return switch (args.encoding) {
+                                        .buffer => .{
+                                            .result = .{
+                                                .buffer = Buffer.fromBytes(bun.default_allocator.dupe(u8, file.contents) catch @panic("out of memory"), bun.default_allocator, .Uint8Array),
+                                            },
+                                        },
+                                        else => brk2: {
+                                            if (comptime string_type == .default) {
+                                                break :brk2 .{
+                                                    .result = .{
+                                                        .string = bun.default_allocator.dupe(u8, file.contents) catch @panic("out of memory"),
+                                                    },
+                                                };
+                                            } else {
+                                                break :brk2 .{
+                                                    .result = .{
+                                                        .null_terminated = bun.default_allocator.dupeZ(u8, file.contents) catch @panic("out of memory"),
+                                                    },
+                                                };
+                                            }
+                                        },
```

Comment:
would the following look better?
```zig
else => if (comptime string_type == .default)
    .{
        .result = .{
            .string = bun.default_allocator.dupe(u8, file.contents) catch @panic("out of memory"),
        },
    }
else
    .{
        .result = .{
            .null_terminated = bun.default_allocator.dupeZ(u8, file.contents) catch @panic("out of memory"),
        },
    },

Reply:
nested switch used to have bugs, but maybe they fixed it